Add team arrival checklist

RealOrangeOne opened the pull request:
Form recreated from existing arrivals checklist doc. I intentionally removed the year specifier, people can just add the TLA rows in locally and render them if necessary, so there's less to change.

Original: [arrival_check_list.zip](https://github.com/srobo/kit-coordination-docs/files/2985772/arrival_check_list.zip) (A `.zip` because GitHub doesn't let you upload `.odt` :-1: )

RealOrangeOne committed:
Add team arrival checklist

RealOrangeOne committed:
Add arrival checklist to artifacts

PeterJCLaw commented on team-arrival-checklist/form.tex:
```diff
@@ -0,0 +1,38 @@
+\begin{table}[h]
+\large
+\def\arraystretch{1.5}
+\begin{tabularx}{\textwidth}{|Y|Y|Y|}
```
This table doesn't have a top line, not sure if that's intentional.

RealOrangeOne replied:
It does, there's a `\hline` after the `\begin{tabularx}` command.

PeterJCLaw replied:
There's an `\hline` after the first (headings) row, but not one before it. This means that the top of the header row is missing a line:

![2019-03-24-223213_903x255_scrot](https://user-images.githubusercontent.com/336212/54886866-bd4b7780-4e84-11e9-8ead-1bbb21bbae0a.png)

PeterJCLaw commented on team-arrival-checklist/form.tex:
```diff
@@ -0,0 +1,38 @@
+\large
+\def\arraystretch{1.5}
+\begin{tabularx}{\textwidth}{|Y|Y|Y|}
+    \textbf{TLA} & \textbf{Time of Arrival} & \textbf{Special Requirements?} \\
```
Why are we capturing the time of arrival? (I'm not sure how this sheet fits into the reception flow)

RealOrangeOne replied:
It's just to keep track of these things. I agree it's not necessarily a useful metric to keep, a simple tick would suffice. It may be useful in future for _something_.

PeterJCLaw replied:
I'm pretty sure we could work out the time of arrival from the reception stuff -- it captures the date at which each username was scanned as present at the competition. Obviously that only applies for the Saturday though -- is the plan to use this on both days?

RealOrangeOne replied:
We probably should, although this form doesn't have anywhere to put the date its for, which probably isn't ideal. Will amend.

RealOrangeOne committed:
Add date to arrivals checklist

PeterJCLaw commented on team-arrival-checklist/Makefile:
```diff
@@ -0,0 +1,6 @@
+all: form.pdf
+
+
+form.pdf: form.tex
```
Could we name this more uniquely?

PeterJCLaw commented on team-arrival-checklist/form.tex:
```diff
@@ -0,0 +1,42 @@
+When a team arrives, please check them off on this sheet by filling in their time of arrival.
+\bigskip
+
+{\large{Date}: \hrulefill}
```
Just listing other things we might want to capture here. Feel free to ignore if you've already considered these:
- the volunteer on reception
- which reception desk it was

I'm not actually sure how useful these would be, but they're pretty easy to capture and might be handy. Super low priority to add though and I'm happy to omit them entirely.

RealOrangeOne replied:
I think it's ok to leave these out, i'm not sure how useful they'll be. If we do need to chase something up, the schedule can tell us who to poke.

RealOrangeOne committed:
Name PDF more uniquely

PeterJCLaw reviewed (approved):
Have you checked what this looks like with lots of teams on the page? (does it wrap suitably onto a second page?)

Otherwise seems good

RealOrangeOne commented:
> does it wrap suitably onto a second page

Hmm, after testing, apparently not. Stand by for fix.

RealOrangeOne committed:
Make table wrap across pages

This unfortunately limits the page to bring narrower and in the middle,
but that's not a massive issue

RealOrangeOne commented:
Fixed, although the table is now narrower (still plenty wide enough to fix what we need). 

I welcome someone with more time / LaTeX skills than myself to fix this, but for now, I think this is perfectly fine!

PeterJCLaw commented on team-arrival-checklist/team-arrival-checklist.tex:
```diff
@@ -0,0 +1,35 @@
+\large
+\def\arraystretch{1.5}
+\begin{tabularx}{\textwidth}{|c|c|c|}
+    \textbf{TLA} & \textbf{Time of Arrival} & \textbf{Special Requirements?} \\
```
not sure if intentional: we're missing a top line for this table.

RealOrangeOne replied:
Not intentional, I spent about 30 seconds trying to put one in, it kept failing, so I left it. I don't think it's requited for this.

PeterJCLaw commented on team-arrival-checklist/team-arrival-checklist.tex:
```diff
@@ -0,0 +1,35 @@
+\begin{tabularx}{\textwidth}{|c|c|c|}
+    \textbf{TLA} & \textbf{Time of Arrival} & \textbf{Special Requirements?} \\
+\hline
+ABC & & \\ \hline
```
What's the plan for adding more teams? Just doing it by hand?

RealOrangeOne replied:
For now, yes. We can audomate it later

PeterJCLaw reviewed (approved):
Heh, or the alternative is to just generate two PDFs each of a single page, that way we get the heading on both pages too.

Ideally I think we'd have a script which injected the TLAs, but for this year manually will do. Let's get this in.
